pkg/plugins/resources/k8s/native/controllers/doc.go

```diff
@@ -14,3 +14,5 @@ package controllers
 // +kubebuilder:rbac:groups=kuma.io,resources=retries,verbs=get;list;watch
 
 // +kubebuilder:rbac:groups=kuma.io,resources=proxytemplates,verbs=get;list;watch;create;update;patch;delete
+
+// +kubebuilder:rbac:groups=kuma.io,resources=gateways;gatewayroutes,verbs=get;list;watch;create;update;patch;delete
```

Comment:
nit: suggest we do one type per line, since that's consistent with the other definitions here, and makes it easier to see there's multiple types